docs/recipes/vite-rspack-webpack-mf.mdx

```diff
@@ -0,0 +1,330 @@
+# Vite + Webpack + Rspack with Module Federation
+
+Vite is the first build tool Zephyr is able to handle Module Federation configuration directly. This guide aims to walk you through how you can deploy a Micro-Frontend application using the Official Vite Plugin from Module Federation. After this guide, you will have a React application consuming remote applications bundled by Vite, Rspack and Webpack deployed through Zephyr Cloud.
+
+:::info Prerequisites
+
+- We expect you to have finished our [Get Started](/general/get-started) guide.
+- Have our [browser extension](https://chromewebstore.google.com/detail/zephyr-mission-control/liflhldchhinbaeplljlplhnbkdidedn) installed in your browser.
+- A registered account on [Zephyr Cloud](https://app.zephyr-cloud.io).
+- npm >=10
+- node >=20
+
+If you don't have `nvm` to manage your node version yet, head to [the official nvm guide](https://github.com/nvm-sh/nvm) to install it.
+
+```bash
+nvm use 20 
+```
+
+:::
+
+### Install Zephyr Plugins
+
+For applications built with Vite: 
+
+```sh title="Terminal"
+npm i vite-plugin-zephyr@latest 
+```
+
+For applications built with Webpack and Rspack: 
+
+```sh title="Terminal"
+npm i zephyr-webpack-plugin@latest
+```
+
+## Example configuration
+Four example build configuration for Vite, Rspack and Webpack. 
```

Comment:
For example build

Reply:
https://github.com/ZephyrCloudIO/zephyr-documentation/pull/27/commits/109c740762d8a4f833b0aa9f1a41810e592013fe